2216/2/versus.rb

```diff
@@ -0,0 +1,91 @@
+require 'optparse'
+require 'russian_obscenity'
+require 'unicode'
+require_relative 'badwordsanalizator'
```

Comment:
I wonder how this would work taking into account that you've changed the file names..